docs/infrastructure/deployment-targets/cloud-target-discovery/index.md

```diff
@@ -24,7 +24,25 @@ The variables used are different for each supported cloud provider.
 
 ### Azure
 
-To discover Azure cloud resources, Octopus uses an [Azure account](/docs/projects/variables/azure-account-variables.md) variable named **Octopus.Azure.Account**.
+To discover Azure cloud resources, Octopus uses the following variables:
+
+| Name                   | Required | Description                                                                                                   |
+| ---------------------- | -------- | ------------------------------------------------------------------------------------------------------------- |
+| Octopus.Azure.Accounts | Y        | An [Azure account](/docs/projects/variables/azure-account-variables.md) to use when discovering cloud targets |
```

Comment:
Is the plural intentional on the variable name?
```suggestion
| Octopus.Azure.Account | Y        | An [Azure account](/docs/projects/variables/azure-account-variables.md) to use when discovering cloud targets |
```

Reply:
Ah nice pick up, that should be plural, thanks!